modules/users/client/views/profile/profile-view-basics.client.view.html

```diff
@@ -15,6 +15,23 @@ <h4 class="profile-username">@{{ profileCtrl.profile.displayUsername || profileC
       <span ng-if="profileCtrl.profile.gender" ng-class="{ 'text-capitalize': !profileCtrl.profile.birthdate }">{{ profileCtrl.profile.gender }}.</span>
     </div>
 
+    <!-- Reply Rate -->
+    <div class="profile-sidebar-section">
+      <i class="icon-fw icon-reply text-muted"></i>
+      <span ng-if="profileCtrl.profile.replyRate">
+        Replied to
+        <span ng-bind="profileCtrl.profile.replyRate"></span>
+      </span>
+      <span ng-if="profileCtrl.profile.replyTime">
+        in
+        <span ng-bind="profileCtrl.profile.replyTime"></span>
+        on average
+      </span>
+      <span ng-if="!profileCtrl.profile.replyRate && !profileCtrl.profile.replyTime">
+        Reply rate not available
+      </span>
```

Comment:
done.

(Seems that `::(profileCtrl.profile.replyRate || profileCtrl.profile.replyTime)` is necessary.)

(btw fyi my experience in angular is low)